pkgs/top-level/aliases.nix

```diff
@@ -64,6 +64,8 @@ doNotDisplayTwice rec {
   lttngTools = lttng-tools;  # added 2014-07-31
   lttngUst = lttng-ust;  # added 2014-07-31
   manpages = man-pages; # added 2015-12-06
+  man_db = man-db; # added 2016-05
```

Comment:
Thanks. I looked at that but forgot to change it in the end. New commit 3472b61.